compose/foundation/foundation/src/desktopMain/kotlin/androidx/compose/foundation/internal/ClipboardUtils.desktop.kt

```diff
@@ -19,41 +19,59 @@ package androidx.compose.foundation.internal
 import androidx.compose.ui.platform.ClipEntry
 import androidx.compose.ui.platform.NativeClipboard
 import androidx.compose.ui.text.AnnotatedString
+import java.awt.datatransfer.Clipboard
+import java.awt.datatransfer.ClipboardOwner
 import java.awt.datatransfer.DataFlavor
-import java.awt.datatransfer.StringSelection
+import java.awt.datatransfer.Transferable
+import java.awt.datatransfer.UnsupportedFlavorException
 import java.io.IOException
-import kotlinx.coroutines.Dispatchers
-import kotlinx.coroutines.withContext
+
+private val _annotatedStringFlavor: DataFlavor =
+    DataFlavor(AnnotatedString::class.java, "AnnotatedString")
+
+@Suppress("UnusedReceiverParameter")
+val DataFlavor.annotatedStringFlavor: DataFlavor
```

Comment:
A public API change require running `./gradlew :mpp:jbApiDump` and commiting the change.

Reply:
cc @igordmn, Igor adding you as a reviewer as here we have an API change for desktop.

Reply:
Done, API dump updated!

Reply:
For context, this is necessary for users to be able to use the `Transferable#isDataFlavorSupported(DataFlavor)` and `Transferable#getTransferData(DataFlavor)` APIs. Otherwise, they'd have to guess what the flavour to use is...

Reply:
I gave it another thought.

IMO, It would be better to move a new public property to ui module, in PlatformClipboard.desktop.kt, so the new property won't expose an extra public class `ClipboardUtils_desktopKt` (we can see it now in foundation.api dump).   

And adding `@ExperimentalComposeUiApi` to it would be safer for now.

Reply:
Happy to do if Igor agrees

Reply:
If it is okay that it will definitely break, we can merge it as is, though, let's add a comment for `private val annotatedStringFlavor: DataFlavor`:
```
This implementation detail is used by Jewel. When remove it, please provide an alternative of retrieving an annotated string, and notify a Jewel developer that they need to change the implementation.
```